packages/swing-store/src/swingStore.js

```diff
@@ -273,8 +300,12 @@ export function makeSwingStoreExporter(dirPath, exportMode = 'current') {
  *   to metadata kept in the kvStore.  Persistently stored in a sqllite table.
  *
  * snapStore - large object store used to hold XS memory image snapshots of
- *   vats.  Objects are stored in files named by the cryptographic hash of the
- *   data they hold, with tracking metadata kept in the kvStore.
+ *   vats.  Objects are stored in a separate table keyed to the vat and delivery
+ *   number of the snapshot, with tracking metadata kept in the kvStore.
+ *
+ * bundleStore - large object store used to hold JavaScript code bundles.
+ *   Objects are stored in a separated table keyed by bundleID, with tracking
```

Comment:
maybe mention that we only accept bundles with `{ moduleFormat: 'EndoZipBase64' }`, and that we deconstruct them and only store the binary Zip-format data in the DB
 
I think the second "Objects are stored" line might be a cut-and-paste duplicate, or should say "bundle contents are stored.." or something. And I don't think "tracking metadata kept in the kvStore" is true here, or at least it's not a concern of swing-store (the kernel has kvStore entries to remember which vats are built from which bundles, but the swingstore doesn't know that)

Reply:
(FYI, I'll be landing a change soon after this PR lands, which accepts `nestedEvaluate` -format bundles too, using a different DB format)